### ISSUE TITLE
Colors not rendered over ssh for specific terminal emulators

### ISSUE THREAD

**Issue by SufyanDahalan** [charmbracelet/bubbletea#730]
**Describe the bug**

**Setup**
Please complete the following information along with version numbers, if applicable.
 - OS NixOS 22.11
 - Shell zsh, bash
 - Terminal Emulator alacritty, xterm
 - Terminal Multiplexer tried once with zellij and once without a multiplexer

**To Reproduce**
1. connect to a machine via ssh
2. Run example/list-simple or example/list-fancy
3. Colors (purple in this case) are not rendered

**Source Code**
examples of this commit hash `29254a08f1458a1ba43db4d2b902982f10758c71`

**Expected behavior**
Colors get rendered

**Screenshots**
![image](https://user-images.githubusercontent.com/76756757/235642825-ef2ea16e-63b4-4828-b950-7e979bef74e5.png)

**Additional context**
The local TTY works correctly. The colored shell prompt on the remote machine also work correctly.
go version go1.19.3 linux/amd64, on both local and remote machine.
Using Konsole or gnome-terminal terminal emulator colors get rendered correctly, looks like the problem is tied to alacritty+ssh

**Comment by muesli** [comment]
What are the values of your `TERM` and `COLORTERM` env vars?

**Comment by SufyanDahalan** [comment]
ssh+Konsole (working): `TERM`="xterm-256color", `COLORTERM` not set
ssh+alacritty (not working): `TERM`="alacritty", `COLORTERM` not set

I will set my TERM to xterm-256colors in my config, this seems to fix it for zsh.

I noticed however that this does'nt fix the issue for powershell 7.2.4. Tried it twice, once with `TERM` set to `screen-256color`, another time with `TERM`="xterm-256color". Both times in Konsole
